src/frontend/stories/draft-index.vue

```diff
@@ -26,11 +26,15 @@
       <ContentSidebar>
         <template #meta-box>
           <MetaBox
-            :created-at="props.draft.createdAt"
-            :updated-at="props.draft.updatedAt"
-            :story-type="props.meta.storyType"
-            :chapter-type="metaChapter"
-            :published-when="publishedWhen"
+            :primary="[
+              { label: props.meta.storyType, value: props.storyName },
```

Comment:
Also here and in the rest of the template:
```suggestion
              { label: meta.storyType, value: storyName },
```